kafka/README.md

```diff
@@ -145,7 +145,7 @@ This object contains information about the message representation in Kafka.
 
 Field Name | Type | Description
 ---|:---:|---
-<a name="messageBindingObjectKey"></a>`key` | [Schema Object][schemaObject] \| [Reference Object](referenceObject) \| [AVRO Schema Object](https://avro.apache.org/docs/current/spec.html) | The message key. **NOTE**: You can also use the [reference object](https://asyncapi.io/docs/specifications/v2.4.0#referenceObject) way.
+<a name="messageBindingObjectKey"></a>`key` | [Schema Object][schemaObject] \| [Reference Object](referenceObject) \| [AVRO Schema Object](https://avro.apache.org/docs/current/spec.html) | The message key. **NOTE**: You can also use the [reference object](referenceObject) way.
```

Comment:
Serious question: Shouldnt you add a slash prefix for anchors? 

```suggestion
<a name="messageBindingObjectKey"></a>`key` | [Schema Object][schemaObject] \| [Reference Object](referenceObject) \| [AVRO Schema Object](https://avro.apache.org/docs/current/spec.html) | The message key. **NOTE**: You can also use the [reference object](#referenceObject) way.
```

Reply:
Then the rest of the references are incorrect as well 🤔 i.e. `[Schema Object][schemaObject] \| [Reference Object](referenceObject)`

Reply:
Seems like they are 🤔

Reply:
Let me fix them in another PR after this one.

Reply:
Nothing is wrong with the anchors @jonaslagoni. I didn't notice those anchors are defined at the very bottom of the document, pointing to the reference in AsyncAPI website so no need for `#` prefix.

However, the links are wrong pointing to 2.4.0 spec reference:

```
[schemaObject]: https://www.asyncapi.com/docs/specifications/2.4.0/#schemaObject
[referenceObject]: https://www.asyncapi.com/docs/specifications/2.4.0/#referenceObject
```